src/modules/renderTasks.js

```diff
@@ -0,0 +1,71 @@
+import itemsArray from './itemsArray.js';
+
+const toDoItemsEl = document.querySelector('.to-do-items');
+const editTaskBtnEl = document.getElementsByClassName('editTaskBtn');
+const editTaskEl = document.getElementsByClassName('to-do-edit');
+const toDoTask = document.getElementsByClassName('to-do-task');
+
+const renderToDoItems = () => {
+  let markup = '';
+
+  itemsArray.forEach((elem, index) => {
+    markup += `<section class="to-do-item  border-bottom" id=${index}>
+    <div class="to-do-edit hide flex-row border-bottom">
+    <input type="text" class="to-do-input" placeholder="Input a task or use the delete button...">
+    <span class="material-symbols-outlined to-do-addTask">add_task</span>
+    </div>
+    <div class="flex-row to-do-task">
+    <input type="checkbox" ${elem.completed && 'checked'}>
+    <p class='description'>${elem.description}</p>
+    <span class="to-do-item-icons">
+    <span class="material-symbols-outlined dltTaskBtn" id=${index}>delete</span>
+    <span class="material-symbols-outlined editTaskBtn" id=${index}>edit_note</span>
+    </span>
+    </div>
+  </section>`;
+  });
+
+  const removeTask = () => {
+    const removeBtnsEl = [...document.getElementsByClassName('dltTaskBtn')];
+    removeBtnsEl.forEach((item) => {
+      item.addEventListener('click', (e) => {
+        itemsArray.splice(e.target.id, 1);
+        localStorage.setItem('itemsArray', JSON.stringify(itemsArray));
+        renderToDoItems();
+      });
+    });
```

Comment:
- [ ] As requested previously remember to update the indexes of the task when a task is deleted. So for example, if I have **`three`** tasks with the `index` properties: `1`, `2`, `3`, If I delete the second task (with `index: 2`), the remaining tasks should be updated to have index `1` and `2`, not `1` and `3`

  ```
  [{description: 'first task', completed: false, index: 1},
    {description: 'second task', completed: false, index: 2},
    {description: 'third task', completed: false, index: 3}]
  ```
  When I remove second task the new array should be:
  ```
  [{description: 'first task', completed: false, index: 1},
    {description: 'third task', completed: false, index: 2}]
  ```